packages/paste-website/src/pages/getting-started/about-paste/index.mdx

```diff
@@ -50,7 +50,7 @@ Paste is a design system used to build accessible, cohesive, and high-quality cu
 | DSYS                            | The team that makes this design system                                                                                                                                                                                                                                                                                                                                                                                     |
 | [Tokens](/tokens)               | Named entities that store visual design attributes                                                                                                                                                                                                                                                                                                                                                                         |
 | [Primitives](/primitives)       | Unstyled, accessible UI interaction or behavior that can be used as a starting point for specific UI needs                                                                                                                                                                                                                                                                                                                 |
-| [Components](/components)       | A suite of common UI components (in Sketch and React) that can be composed into any web based user experience. They follow the WCAG 2.1 AA standard and the W3C ARIA authoring practices to ensure the best accessibility practices are baked into them. They are available to be installed via NPM or Abstract for anyone to use.                                                                                         |
+| [Components](/components)       | A suite of common UI components (in Figma and React) that can be composed into any web based user experience. They follow the WCAG 2.1 AA standard and the W3C ARIA authoring practices to ensure the best accessibility practices are baked into them. They are available to be installed via NPM or Abstract for anyone to use.                                                                                          |
```

Comment:
Are we still using Abstract, or should this say "available to be installed via NP or _Figma_ for anyone to use"?

Reply:
Good catch. I'll change that.